deploy/main.tf

```diff
@@ -1,18 +1,21 @@
-// Resource group configuration
+data "local_file" "script" {
+  filename = "../src/List-PastAlerts.ps1"
+}
+
+data "azurerm_subscription" "primary" {
+}
 
-resource "azurerm_resource_group" "rg-monitor-automation" {
+resource "azurerm_resource_group" "rg" {
   name     = var.rg_name
   location = var.rg_location
 
   tags = local.tags
 }
 
-## Automation account configuration
-
-resource "azurerm_automation_account" "aa-monitor-automation" {
+resource "azurerm_automation_account" "account" {
```

Comment:
> Do not repeat resource type in resource name (not partially, nor completely)

Either `this` or the purpose the resource has. Do not include `automation` or `aa` etc

Reply:
Fixed